questions/services/multiple_choice_handlers.py

```
@@ -0,0 +1,376 @@
+
+def multiple_choice_rename_option(
+    question: Question,
+    old_option: str,
+    new_option: str,
+) -> Question:
+    """
+    Modifies question in place and returns it.
+    Renames multiple choice option in question options and options history.
+    """
+    if question.type != Question.QuestionType.MULTIPLE_CHOICE:
+        raise ValueError("Question must be multiple choice")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)